content/05.dl-complexities.md

```diff
@@ -1 +1,25 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with
+different tweaks and modifications.
+Despite improved framework ease-of-use and on-demand cloud computing resources
+this means DL can be very demanding, requiring significant
+infrastructure and patitence to complete.
+The experimentation inherent to DL is often noisy (requiring repetition) and
+represent a significant organisational challenge.
```

Comment:
"organisational" => "organizational"

(sorry, don't want to be nit-picky, but we used AE elsewhere)

Reply:
Not at all, totally automatic Britishism on my behalf.